core/domain/stats_services.py

```diff
@@ -16,251 +16,137 @@
 
 """Services for exploration-related statistics."""
 
-import itertools
-
-from core.domain import exp_services
-from core.domain import interaction_registry
+from core.domain import exp_domain
 from core.domain import stats_domain
-from core.domain import stats_jobs_continuous
 from core.platform import models
 
 (stats_models,) = models.Registry.import_models([models.NAMES.statistics])
 
 
-# TODO(bhenning): Test.
-def get_visualizations_info(exploration_id, state_name):
-    """Returns a list of visualization info. Each item in the list is a dict
-    with keys 'data' and 'options'.
+def handle_new_exp_stats_creation(exploration):
```

Comment:
You know, I'm not sure you need the deferred queue at all. Why not just do it inline? I can see why you'd want to defer on the learner side but I'm not sure it makes a lot of sense on the creator side.

Reply:
@seanlip Wasn't the whole point of making this async because we don't want the creator to be kept waiting while committing an exploration. We are already making the commit part of a exploration more time consuming after adding the ML stuff.

Reply:
Nope, I'm not worried about creating or updating an exploration taking some time. It's a big operation and does not happen too frequently. I think it's fine.

Reply:
@seanlip Aha. If thats the case, we can ditch async completely and go with normal methods. The stats functions will add only one GET op and one POST op. So, it shouldn't be much

Reply:
Done.